exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -184,6 +184,17 @@ private ServerMeta getServerMeta() throws SQLException {
     return serverMeta;
   }
 
+  /**
+   * The same as {@link DrillDatabaseMetaDataImpl#getServerMeta()} but updates server metadata for the every call
+   * @return server meta information
+   * @throws SQLException for error when getting server meta
+   */
+  private ServerMeta getUpdatedServerMeta() throws SQLException {
```

Comment:
There's nothing in the JDBC spec which says if this has to be refreshed when user session is changed manually. It seems that MySQL doesn't do a server call each time, but I haven't checked other drivers (not all databases allow to change the quoting identifier in the session).

I would say it might not be necessary as someone who changed manually the quoting know that he/she is dealing with Drill and that he/she changed the the default quoting identifier. But people might have strong arguments in favor of refreshing...